test/NHSD.BuyingCatalogue.Documents.API.Tests/SolutionsControllerTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NHSD.BuyingCatalogue.Documents.API.Controllers;
+using NHSD.BuyingCatalogue.Documents.API.Storage;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Documents.API.Tests
+{
+    internal class SolutionsControllerTests
+    {
+        [Test]
+        public void GetFileNames_ReturnsStorageResult()
+        {
```

Comment:
Are we just checking the dependency is called once here? Just wondering whether or not we should also be checking the result contains some data. Obviously the mock would have to contain some data.